Fix oid and root_oid that might have been accidentally converted to text when a DB was migrated from py2.
[pbauer]